modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -130,11 +130,10 @@ inline IE::TensorDesc toIE(const cv::Mat &mat, cv::gapi::ie::TraitAs hint) {
         const size_t height   = mat.size().height;
         const size_t width    = mat.size().width;
 
-        const size_t strideH  = mat.step[0];
-
-        IE::BlockingDesc bdesc({1, height, width, channels} /* dims */,
```

Comment:
looks like the original code actually is correct in terms of order for "blocked_dims", they should come in actual order NHWC..

Reply:
I agree. BlockedDims already describes the physical dimensions.